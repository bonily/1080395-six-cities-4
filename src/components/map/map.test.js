import React from "react";
import renderer from "react-test-renderer";
import Map from "./map";

const OFFERS = [
  {
    coords: [52.3909553943508, 4.85309666406198],
  },
  {
    coords: [52.369553943508, 4.85309666406198],
  },
  {
    coords: [52.3909553943508, 4.929309666406198],
  },
  {
    coords: [52.3809553943508, 4.939309666406198],
  },
];

describe(`MapSnapTest`, () => {
  it(`Map should render map component with pins`, () => {
    const tree = renderer
      .create(<Map
        offers = {OFFERS}
      />,
      {
        createNodeMock: () => document.createElement(`div`)
      }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

