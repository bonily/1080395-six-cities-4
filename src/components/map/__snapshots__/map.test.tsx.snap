// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapSnapTest Map should render map component with pins 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "width": 512,
    }
  }
>
  
</div>
`;
